CONTRIBUTING.md

```diff
@@ -1 +1,51 @@
-admin-secure-payment-content: "Utilizamos o Stripe, serviço de pagamento já consolidado no mercado, para efetuar as transações com segurança"
+#7
+_Fork is a resource that allows the user to have a copy of a project. The project can have new ramifications after being approved by the admin._ 
```

Comment:
Actually the project have a ramification after a fork is created, and don't need be approved by the admin, you just need to approve when a pull request is created. When you fork you have your own repository copy for you. Please rephrase this.